src/main/java/core/basesyntax/service/impl/ReportCreatorImpl.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.dao.FruitsDao;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.ReportCreator;
+import java.util.List;
+
+public class ReportCreatorImpl implements ReportCreator {
+    private FruitsDao fruitsDao;
+
+    public ReportCreatorImpl(FruitsDao fruitsDao) {
+        this.fruitsDao = fruitsDao;
+    }
+
+    @Override
+    public String createHeader(String firstColumn, String secondColumn) {
+        return firstColumn + "," + secondColumn;
+    }
+
+    @Override
+    public String create(String header) {

```

Comment:
```suggestion
    public String create(Map<String, Integer> data) {
```
you should pass data to this method as a parameter